epub33/core/index.html

```diff
@@ -1213,6 +1213,29 @@ <h3>Exempt resources</h3>
 							CSS style sheets) is an exempt resource.</p>
 					</dd>
 
+					<div class="proposed correction" id="change_1">
+						<span class="marker">Proposed Correction 1</span>
+						<p>The exemption for resources not referenced from the spine or embedded in content documents
+							(refer to the paragraphs following note below), includes resources referenced from
+								<code>script</code> elements, as well as resources retrieved by scripts for processing.
+							The description focused on data sets travelling in the EPUB container, but that was only one
+							use case. This change adds scripting to the major exemption groups and removes it from the
+							case for resources travelling in the container. For more information, refer to <a
+								href="https://github.com/w3c/epub-specs/issues/2649">issue 2649</a>.</p>
+					</div>
+
+					<ins cite="#change_1">
+						<dt id="exempt-script">Scripts and script resources</dt>
+						<dd id="confreq-resources-cd-fallback-script">
+							<p>Any resource referenced from the [[html]] <a data-cite="html#the-script-element"
+										><code>script</code></a> element that is not already a core media type resource
```

Comment:
nit: s/already// for brevity, but you are the wordsmith so final authority is yours :)

Reply:
Ya, I copied the text from exemption above but agree it doesn't add anything. Will take it out of both.